standalone/src/main/scala/filodb.standalone/NewFiloServerMain.scala

```diff
@@ -54,4 +59,29 @@ object NewFiloServerMain extends StrictLogging {
         logger.error("Error occurred when initializing FiloDB server", e)
     }
   }
+
+  private def startGrpcServer(settings: FilodbSettings, coordinatorActor: ActorRef): Unit = {
+      if (settings.config.getBoolean("grpc.start-grpc-service")) {
+        val client = new LocalClient(coordinatorActor)
+        // TODO: Remove hardcoding
+        val dsRef = DatasetRef("prometheus")
```

Comment:
Can these be made configurable in this PR?

Reply:
I couldn't find a good existing property that I can use. I'll leave it for now if thats ok.